tests/mysql_insertion/scripts/http-get-request.jmx

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
should be renamed to http-post-request.jmx
This will be called from run.sh. line 21